README.md

```diff
@@ -1374,12 +1374,12 @@ _You can enable the following settings in Xcode by running [this script](resourc
   * `// MARK: Fileprivate` for `fileprivate` properties and methods.
   * `// MARK: Private` for `private` properties and methods.
   * If the type in question is an enum, its cases should go above the first `// MARK:`.
-  * If there are typealiases or nested types, they should go above the first `// MARK:`.
   * Do not subdivide each of these sections into subsections, as it makes the method dropdown more cluttered and therefore less useful. Instead, group methods by functionality and use smart naming to make clear which methods are related. If there are enough methods that sub-sections seem necessary, consider refactoring your code into multiple types.
   * If all of a type's definitions belong to the same category (e.g. the type only consists of `internal` properties), it is OK to omit the `// MARK:`s.
   * If the type in question is a simple value type (e.g. fewer than 20 lines), it is OK to omit the `// MARK:`s, as it would hurt legibility.
 
 * <a id='subsection-organization'></a>(<a href='#subsection-organization'>link</a>) **Within each top-level section, place content in the following order.** This allows a new reader of your code to more easily find what they are looking for. [![SwiftFormat: organizeDeclarations](https://img.shields.io/badge/SwiftFormat-organizeDeclarations-008489.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#organizeDeclarations)
+  * Nested types
```

Comment:
@calda do you see value in clarifying that tyepaliases go here too (assuming that is correct)?

Reply:
Should be a simple follow up PR. I think the clarity could be nice.

Reply:
Right now the `organizeDeclarations` rule will place `typealias` declarations with the instance properties (https://github.com/nicklockwood/SwiftFormat/blob/master/Sources/Rules.swift#L5072). 

I agree that we should specify exactly where they should go. If we want to place them somewhere else, I can make a follow-up SwiftFormat enhancement.

Reply:
@calda it seems like it would be logic to group them with nested types. I don't feel strongly that they are positioned _before_ or _after_ nested types that are defined inline (rather than typealiased). This seems like it should be _valid_:


```swift
class Foo {

  // MARK: Public

  public enum MyEnum { }

  public typealias Bar = String

  public struct YourStruct { }

  // MARK: Internal
  
  var value = 1
}
```